idaes/core/util/scaling.py

```diff
@@ -72,7 +73,7 @@ def scale_arc_constraints(blk):
             constraint_scaling_transform(c, sf)
 
 
-def map_scaling_factor(iter, default=1, warning=False, func=min):
+def map_scaling_factor(iter, default=1, warning=False, func=min, hint=None):
```

Comment:
Probably should document what `hint` is for - I can guess but it would be good to state it clearly.